docs/api.rst

```diff
@@ -95,33 +98,69 @@ Methods
    count_cohort_alleles
    count_variant_alleles
    count_variant_genotypes
+   observed_heterozygosity
+   variant_stats
+
+Population and Cohort Genetics
+------------------------------
+
+.. autosummary::
+   :toctree: generated/
+
    divergence
    diversity
    Fst
+   Tajimas_D
+   Weir_Goudet_beta
    Garud_H
-   genee
+   pbs
+
+Pedigree and Relationship
+------------------------
+
+.. autosummary::
+   :toctree: generated/
+
    genomic_relationship
-   gwas_linear_regression
-   hardy_weinberg_test
+   pc_relate
+   pedigree_contribution
+   pedigree_inbreeding
+   pedigree_inverse_kinship
+   pedigree_kinship
    hybrid_inverse_relationship
    hybrid_relationship
    identity_by_state
-   individual_heterozygosity
+
+Linkage Disequilibrium and Heterozygosity
```

Comment:
I think just keep a section for LD, and move heterozygosity to "basic genetics" above (there's already one heterozygosity function above)